myapp/segmentsconfigapp/tests.py

```diff
@@ -0,0 +1,43 @@
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from segmentsconfigapp.parser import ParserSegmentConfig
+
+# Create your tests here.
+class AccountTests(APITestCase):
+    def test_parsing(self):
+        test_li = [("@product\n*/product/*\n\n@article\n*article.html\n@comments\n*/comments/*.html",True),#OK
+                             ("@product\n\n\n@article\n*article.html\n@comments\n*/comments/*.html", False),#FAIL
+                             ("@product\n*/product/*\n\n*article.html\n@comments\n*/comments/*.html", False),#FAIL
+                             ("@product\n*/product/*", True),#OK
+                             ("@product\n*/product/*\n@product\nproduct.html", False),#Fail
+                             ]
+        for test in test_li:
```

Comment:
`for test, expected_return in test_li:` that way you wont need to do `test[x]` later  (it's easier to read)